packages/cdk8s/package.json

```
@@ -61,7 +61,7 @@
   "dependencies": {
     "follow-redirects": "^1.11.0",
     "json-stable-stringify": "^1.0.1",
-    "yaml": "^1.7.2"
+    "yaml": "^2.0.0-0"
```

```suggestion
    "yaml": "2.0.0-0"
```
I would strongly recommend against using a `^` range with prerelease versions. `2.0.0-0` should be usable even in production, but I make no guarantees that the API won't change in `2.0.0-1`.

---

I am okay with that

---

Yes, good point. Thanks for the suggestion